charts/osm/README.md

```diff
@@ -161,6 +161,7 @@ The following table lists the configurable parameters of the osm chart and their
 | contour.contour | object | `{"image":{"registry":"docker.io","repository":"projectcontour/contour","tag":"v1.18.0"}}` | Contour controller configuration |
 | contour.enabled | bool | `false` | Enables deployment of Contour control plane and gateway |
 | contour.envoy | object | `{"image":{"registry":"docker.io","repository":"envoyproxy/envoy-alpine","tag":"v1.19.1"}}` | Contour envoy edge proxy configuration |
+| smi.validateTrafficTarget | bool | `true` | Enables validation of SMI Traffic Target |
```

Comment:
Is this is enabled by default and OSM has a requirement of the traffic target being in the destination ns, why and what would be the use case of giving users the option to disable this ? If the user disables this OSM wouldn't take into account the traffic target right ? If so, making this configurable doesn't seem like an option

Reply:
I've made this configurable in the case that another tool or service mesh in the cluster is leveraging SMI Traffic Target and doesn't need this validation. In that scenario, this option will allow the user to turn the validation off.

I will be adding logic in the next PR to ignore invalid traffic targets. This PR is part 1 of #4116 
example scenarios where there are invalid traffic targets:
1. traffic targets exist before upgrading to this version of OSM 
2. traffic targets are created before OSM is installed